src/main/java/linenux/command/AddCommand.java

```diff
@@ -26,21 +31,49 @@ public boolean respondTo(String userInput) {
 
     @Override
     public CommandResult execute(String userInput) {
+        assert userInput.matches(TASK_PATTERN);
+        String argument = extractArgument(userInput);
+        Matcher matcher = Pattern.compile(ARGUMENT_PATTERN).matcher(argument);
+
+        if (!matcher.matches()) {
+            return makeInvalidArgumentResult();
+        }
+
+        String taskName = matcher.group("taskName");
+
+        if (!isTaskNameValid(taskName)) {
+            return makeInvalidArgumentResult();
+        }
+
+        Task task = new Task(taskName);
+        this.schedule.addTask(task);
+
+        return makeResult(task);
+    }
+
+    private String extractArgument(String userInput) {
```

Comment:
Well, the objective of this PR is not to handle this issue. It will be handled when start and end times are implemented.